baldrick/plugins/github_pull_requests_base_branch.py

```diff
@@ -0,0 +1,22 @@
+from baldrick.plugins.github_pull_requests import pull_request_handler
+
+
+# Or do we want to check every push event?
+@pull_request_handler(actions=['opened'])
```

Comment:
Presumably we also need to re-check if you change the base branch on github or the status will always say failed?

Reply:
Good point. Is it "synchronize"? If not, what should I use?

Reply:
I honestly don't know. "synchronize" is any push.

Reply:
Where did this list come from? I don't see any other event that is appropriate:

https://github.com/OpenAstronomy/baldrick/blob/116546da7f5798c01ef88d1425d2e7a9081b7d06/baldrick/plugins/github_pull_requests.py#L19-L27

Reply:
I think "synchronize" is probably the better one here.